api/src/test/java/org/pmiops/workbench/workspaces/WorkspaceServiceTest.java

```diff
@@ -464,7 +475,9 @@ public void enforceFirecloudAclsInRecentWorkspaces() {
             RawlsWorkspaceAccessLevel.NO_ACCESS,
             WorkspaceActiveStatus.ACTIVE);
     workspaceService.updateRecentWorkspaces(sharedWorkspace);
-
+    when(mockWorkspaceAuthService.enforceWorkspaceAccessLevel(
```

Comment:
could you say a little about why this change is needed?

Reply:
I moved class workspaceAuth from import to mockBean i'll try to add meaningful comment

Reply:
ok cool